File: lib/business/AQI.dart
class AQI {
  double cal(double dust, double co, double smoke, double uv) {
    return (dust / 1.66) * 0.9 +
        ((co + smoke) / 2) * 4.84 * 0.05 +
        uv * 1.2 * 0.05;
  }
}

final aqi = AQI();

File: android/app/src/main/kotlin/com/example/o2_mobile/AQI.kt
package com.example.o2_mobile

class AQI {
    companion object{
        fun cal(dust: Double, co: Double, smoke: Double, uv: Double): Double {
            return (dust / 1.66) * 0.9 + ((co.toDouble() + smoke.toDouble()) / 2) * 4.84 * 0.5 + uv.toDouble() * 1.2 * 0.5
        }
    }
}